pyt/vulnerabilities.py

```diff
@@ -73,7 +73,8 @@ def identify_triggers(
     cfg,
     sources,
     sinks,
-    lattice
+    lattice,
+    nosec_lines=set()
```

Comment:
So `identify_triggers(` is never called in tests, so I don't think a default arg is needed.